src/client/actions/user.js

```diff
@@ -67,3 +75,10 @@ export const savePlaces = () => {
       })
   }
 }
+
+export const deletePlace = (value) => {
```

Comment:
This is the thunk, the reducer for the action is called removePlace at #52 above